saas/views/metrics.py

```diff
@@ -166,9 +166,13 @@ def get_context_data(self, **kwargs):
                           "unit": "$", "table": account_table}
         data['customers'] = {"title": "Customers",
                              "table": customer_table, "extra": customer_extra}
-        context.update({"title": "Revenue Metrics",
-            "data": data,
-            "data_json": json.dumps(data, cls=DjangoJSONEncoder)})
+        context.update({
+            "title": "Revenue Metrics",
+            "tables": [
+                {"title": "Amount", "unit": "$", "key": "amount", "active": True},
```

Comment:
I see the unit was already '$' before but it should surely be changed to use the unit associated with the organization.